src/armonik_cli/commands/sessions.py

```diff
@@ -0,0 +1,206 @@
+        sessions = [_clean_up_status(s) for s in sessions]
+        console.get_console().formatted_print(
+            sessions, format=output, table_cols=SESSION_TABLE_COLS
+        )
+
+    console.get_console().print(f"\n{total} sessions found.")
+
+
+@sessions.command()
+@endpoint_option
+@output_option
+@debug_option
+@click.argument("session-id", required=True, type=str, metavar="SESSION_ID")
+@errors.error_handler
+def get(endpoint: str, output: str, session_id: str, debug: bool):
+    """Get details of a given session."""
+    with grpc.insecure_channel(endpoint) as channel:
+        sessions_client = ArmoniKSessions(channel)
+        session = sessions_client.get_session(session_id=session_id)
+        session = _clean_up_status(session)
+        console.get_console().formatted_print(session, format=output, table_cols=SESSION_TABLE_COLS)
+
+
+@sessions.command()
+@endpoint_option
+@click.option(
+    "--max-retries",
+    type=int,
+    required=True,
+    help="Maximum default number of execution attempts for session tasks.",
+    metavar="NUM_RETRIES"
+)
+@click.option(
+    "--max-duration",
+    type=str,
+    required=True,
+    help="Maximum default task execution time (format HH:MM:SS.MS).",
+    metavar="DURATION"
+)
+@click.option(
+    "--priority",
+    type=int,
+    required=True,
+    help="Default task priority.",
+    metavar="PRIORITY"
+)
+@click.option(
+    "--partition",
+    type=str,
+    multiple=True,
+    help="Partition to add to the session.",
+    metavar="PARTITION"
+)
+@click.option(
+    "--default-partition",
+    type=str,
+    default="default",
+    show_default=True,
+    help="Default partition.",
+    metavar="PARTITION"
+)
+@click.option(
+    "--application-name",
+    type=str,
+    required=False,
+    help="Default application name.",
+    metavar="NAME"
+)
+@click.option(
+    "--application-version",
+    type=str,
+    required=False,
+    help="Default application version.",
+    metavar="VERSION"
+)
+@click.option(
+    "--application-namespace",
+    type=str,
+    required=False,
+    help="Default application namespace.",
+    metavar="NAMESPACE"
+)
+@click.option(
+    "--application-service",
+    type=str,
+    required=False,
+    help="Default application service.",
+    metavar="SERVICE"
+)
+@click.option(
+    "--engine-type",
+    type=str,
+    required=False,
+    help="Default engine type.",
+    metavar="ENGINE_TYPE"
+)
+@click.option(
+    "--option",
+    type=str,
+    required=False,
+    multiple=True,
+    help="Additional default options.",
+    metavar="NAME=VALUE"
+)
+@output_option
+@debug_option
+@errors.error_handler
+def create(endpoint: str, max_retries: int, max_duration: str, priority: int, partition: List[str], default_partition: str, application_name: str | None, application_version: str | None, application_namespace: str | None, application_service: str | None, engine_type: str | None, option: List[str] | None, output: str, debug: bool):
```

Comment:
line too long and no docstring. There are A LOT of parameters. It will be better to document them